src/Corporation/Division.ts

```diff
@@ -836,7 +818,7 @@ export class Division {
           const desiredSellAmount = product.cityData[city].desiredSellAmount;
           if (isString(desiredSellAmount)) {
             //Sell amount is dynamically evaluated
-            let tmp: number | string = desiredSellAmount.replace(/MAX/g, (adjustedQty + "").toUpperCase());
+            let tmp: number | string = desiredSellAmount.replace(/MAX/g, adjustedQty.toString());
```

Comment:
You changed this for products, but not for materials.

Reply:
yeah i wasnt sure about that one didnt know if there was a functional difference between +"").toUpperCase and .toString()

Reply:
Oh there definitely is, but I'm pretty sure it's only things we don't want.